PWGHF/TableProducer/candidateSelectorLcToK0sP.cxx

```diff
@@ -185,6 +302,13 @@ struct HfCandidateSelectorLcToK0sP {
         continue;
       }
 
+      if (applyML) {
+        if (!selectionML(candidate, bach)) {
```

Comment:
Why don't you use one `if`?

Reply:
I was not sure if `&&` guarantees that the right part never gets called if the left evaluates to 'false'. Fixed now as you suggested

Reply:
Yes, it does. Thanks